source/default-conf.py

```diff
@@ -284,7 +284,8 @@
 # k8s is temporary until integrating the references here
 
 intersphinx_mapping = {
-    'baremetal': ('https://www.min.io/docs/minio/', None),
+    'baremetal': ('https://www.min.io/docs/minio/linux/', None),
```

Comment:
baremetal -> linux

we're mostly leaving 'baremetal' behind.

Where this comes into play is, I think, you can prefix a reference to let Sphinx know where to look.

so

:linux:ref:`foo` should specifically look up the `foo` reference for the `linux` intersphinx inventory.

I think.

Reply:
Ah, that makes sense. Fixing.